proposals/lip-0045.md

```diff
@@ -935,41 +804,27 @@ createTerminatedStateAccount(chainID, stateRoot = None):
         }
 
     create an entry in the terminated state substore with
-        storeKey = uint32be(chainID)
-        storeValue = serialization of terminatedState
-
-    return True
+        storeKey = chainID
+        storeValue = encode(terminatedStateSchema, terminatedState)
 ```
 
 #### createTerminatedOutboxAccount
 
 The `createTerminatedOutboxAccount` function creates an entry in the terminated outbox substore.
 
-##### Parameters
-
-* `chainID`: The ID of the chain.
-* `outboxRoot`: The outbox root to be stored in the terminated outbox account.
-* `outboxSize`: The outbox size to be stored in the terminated outbox account.
-* `partnerChainInboxSize`: The partner chain inbox size to be stored in the terminated outbox account.
-
-##### Returns
-
-A boolean indicating the success of the function execution.
-
-##### Execution
-
 ```python
-createTerminatedOutboxAccount(chainID, outboxRoot, outboxSize, partnerChainInboxSize):
+def createTerminatedOutboxAccount(chainID: ChainID, 
```

Comment:
Here and in other places, could we follow the [PEP8 guidelines on indentation](https://legacy.python.org/dev/peps/pep-0008/#indentation) as mentioned in the pseudocode document. In particular, the following is explicitly discouraged in PEP8:

```
# Further indentation required as indentation is not distinguishable.
def long_function_name(
    var_one, var_two, var_three,
    var_four):
    print(var_one)
```

Reply:
fixed